src/utils/formatRelativeTime.js

```diff
@@ -0,0 +1,16 @@
+import { intlFormatDistance } from "date-fns";
+
+const localeMap = {
```

Comment:
I'm lacking a bit of context here, but are we redefining this list, rather than using a shared source of truth?

Reply:
I think the answer is that this map is for ensuring that our locales match up with the ones from the library

Reply:
could we move this into a central location, maybe a separate util called `localeMappings.js` or something, and make sure nowhere else is redefining it?